css/properties/scroll-margin-bottom.json

```diff
@@ -30,18 +30,28 @@
             "opera_android": {
               "version_added": "48"
             },
-            "safari": {
-              "version_added": "11",
-              "alternative_name": "scroll-snap-margin-bottom",
-              "partial_implementation": true,
-              "notes": "Scroll margin is not applied for scrolls to fragment target or <code>scrollIntoView()</code>, see <a href='https://webkit.org/b/189265'>bug 189265</a>."
-            },
-            "safari_ios": {
-              "version_added": "11",
-              "alternative_name": "scroll-snap-margin-bottom",
-              "partial_implementation": true,
-              "notes": "Scroll margin is not applied for scrolls to fragment target or <code>scrollIntoView()</code>, see <a href='https://webkit.org/b/189265'>bug 189265</a>."
-            },
+            "safari": [
+              {
+                "version_added": "14.1"
+              },
+              {
+                "version_added": "11",
+                "alternative_name": "scroll-snap-margin-bottom",
+                "partial_implementation": true,
+                "notes": "Scroll margin is not applied for scrolls to fragment target or <code>scrollIntoView()</code>, see <a href='https://webkit.org/b/189265'>bug 189265</a>."
```

Comment:
It's not strictly required to be addressed, but this data raises some interesting questions:

Does the bug noted apply to standard name? Or are they separate implementations?

The linked bug is fixed, so the answers here might help decide the disposition of the note and partial implementation, depending on whether (and, if so, when) the fixed shipped.

(It looks like this applies to all the `scroll-margin-*` features.)

Reply:
@apple-web-evangelist After _a lot_ of scrolling, I think this is the only actionable unresolved comment standing. Did you plan on tackling this? And do you have any other changes to this PR planned?

Reply:
I see what you did there. 😉

The bug *is* fixed and shipped with 14.1/14.5. I added the additional support for the other `scroll-margin-*` entries in 90183cb. I believe that means we should add the `version_removed` entry in `scroll-margin-bottom`. Is that the last detail to address?

Reply:
Addressed in fbf2d54.

Reply:
Great, the bug being fixed simplifies things. Do I understand correctly that `scroll-snap-*` were all removed? If so, then I think you need to apply the `version_removed` change to the other `scroll-snap-*` statements and then we're done.

(what I wrote before I saw your latest change)

---
If `scroll-snap-*` still exist (that is, the alternative names are still supported), then the notes starting with "Scroll margin is not applied" need to qualify the versions they apply to (i.e., find and replace your way to "Before version 14.1/14.5, scroll margin is…") and then we're done.

If `scroll-snap-*` have been removed, then add a `version_removed` line to those support statements.

---

Either way, I believe that's the last of it. If you can get to that shortly, I can include this in today's release.

Reply:
The `scroll-snap-*` exist as an alias, so the behavior is corrected.

Reply:
In other words, they still work, and the "Scroll margin is not applied…" does not, in fact, apply to those cases.

Reply:
Ah, in that case, I'd suggest undoing fbf2d54 and changing the notes to start with, "Before version 14.1, scroll margin is" (or 14.5, for the iOS entries). Otherwise, it looks like the aliases have gone away.

Reply:
Will do!

Reply:
Addressed in 20dfeee.

Reply:
Thanks, @apple-web-evangelist. Do you want to apply that change to the text like this in the other `scroll-snap` entries too? (that is, `scroll-margin{,-top,-left,-right}.json`)? It's the same bug.

Reply:
Ahh, I missed that. I will correct it quickly.

Reply:
Addressed all of it with 8fc447d.

Reply:
Great. Marking this resolved. 👍